ui-tests/tests/toc.test.ts

```diff
@@ -36,7 +36,7 @@ describe('Table of Contents', () => {
   });
 
   test('Refresh File Browser', async () => {
-    await galata.filebrowser.refresh();
+    await expect(galata.filebrowser.refresh()).resolves.toBeUndefined();
```

Comment:
That `toBeUndefined` looks a bit out of place, but maybe that's something we should address in Galata since some of the functions return a `boolean` (or `Promise<boolean>`) and others don't.

Reply:
Yep, although on that case, it will require so reflection how to check it was done.